packages/gateway/src/repositories/memory.ts

```diff
@@ -93,10 +92,15 @@ export class InMemoryRepository {
     })
   }
 
-  async getDomain({ node }: DomainProps): Promise<Domain | null> {
+  async getDomain({ node }: NodeProps): Promise<Domain | null> {
     return this.domains.get(node) || null
   }
 
+  async getSubdomains({ node }: NodeProps): Promise<Domain[]> {
+    console.log({ node })
```

Comment:
Is there something to be implemented here and in the functions below rather than `console.log`?